vue/src/core/scss/main.scss

```diff
@@ -0,0 +1,76 @@
+.page-header {
+  display: flex;
+  align-items: center;
+  padding: 1rem;
+  background-color: #FFF;
+  box-shadow: 1px 2px 4px rgba(0, 0, 0, 0.1);
+
+  //@media (max-width: 576px){
+  //  padding: 0;
+  //}
+
+  .nav-pills {
+    box-shadow: none;
+    flex-wrap: nowrap;
+  }
+
+  .nav-link {
+    padding-top: 1.1rem;
+    padding-bottom: 1.1rem;
+  }
+
+  @media (max-width: 576px) {
+    .nav-link {
+      padding-left: 0.5rem;
+      padding-right: 0.5rem;
+    }
+  }
+
+  h1 {
+    font-size: 1.5rem;
+    margin-bottom: 0;
+    flex: 1;
+    white-space: nowrap;
+  }
+
+  .breadcrumb {
+    flex: 1;
+    background: transparent;
+    margin: 0;
+  }
+
+  .page-header-fixed.menu-fixed & {
+    left: $menu-width;
+  }
+
+  @include media-breakpoint-down(sm) {
+    .page-header-fixed.menu-fixed & {
+      left: 0;
+    }
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-direction: row;
+
+  }
+
+  .page-header-fixed.menu-fixed.menu-hidden & {
+    left: 0;
+  }
+}
+
+
+.content-spinner {
```

Comment:
@skavt If `ContentSpinner `component is implement in this branch, please use it here and in all places where b-spinner is used. Also do remove .content-spinner css.